### PR TITLE
Bug fixing crash when compile with !use_frameworks in Podfile

### PR DESCRIPTION
**Pull request by rnaharro**
I just created an UIImage category inside the DAAppViewCell.m file to handle the load of images from Main Bundle (compiled as static library) or Framework (compiled as framework).

This could close the #25 issue, created by @stsandro.

### CODE REVIEW

**Comment by coryalder** [comment]
bump! just ran into this issue. :)

File: DAAppsViewController/DAAppViewCell.m [hunk 1]
```diff
@@ -42,7 +61,12 @@ + (void)initialize
         
         NSInteger numberOfStars = 11;
         NSMutableArray *starRatingImages = [[NSMutableArray alloc] initWithCapacity:numberOfStars];
-        UIImage *starsImageSheet = [UIImage imageNamed:@"DAAppsViewController.bundle/DAStarsImage"];
+        UIImage *starsImageSheet = [UIImage imageNamedFromMainBundleOrFramework:@"DAAppsViewController.bundle/DAStarsImage"];
+        if (!starsImageSheet) {
```
Comment (ricsantos) [inline DAAppsViewController/DAAppViewCell.m]: This nil check is redundant as it's taken care of in the category method `imageNamedFromMainBundleOrFramework` (assume it was accidentally left after refactoring).

**Review by ricsantos — CHANGES_REQUESTED** [review]
Nice work, tested and works fine. Just that redundant nil check to fix.

File: DAAppsViewController/DAAppViewCell.m [hunk 2]
```diff
@@ -12,6 +12,25 @@
...
+
+@implementation UIImage (DAAppViewCell)
+
++ (UIImage *)imageNamedFromMainBundleOrFramework:(NSString *)name
+{
+    UIImage *image = [UIImage imageNamed:name];
+    if (!image) {
+        // Try to load from bundle
+        NSBundle *bundle = [NSBundle bundleForClass:[DAAppViewCell class]];
+        image = [UIImage imageNamed:name inBundle:bundle compatibleWithTraitCollection:nil];
```
Comment (ricsantos) [inline DAAppsViewController/DAAppViewCell.m]: This is only available in iOS 8.0 or newer. This pod specifies min target of iOS 5.0 in the readme, but the podspec doesn't actually specify a min target.

**Pull request closed without merging**